cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+    "baseUrl": "http://localhost:3001",
+    "waitForAnimations": true,
+    "animationsDistanceTreshold": 20,
+    "defaultCommandTimeOut": 5000,
+    "execTimeout": 60000,
+    "pageLoadTimeout": 60000,
+    "requestTimeout": 15000,
+    "reponsTimeout": 15000,
+    "failOnStatusCode": false
+}
```

Comment:
Those seem to be Windows carriage returns and should be removed (they are in multiple files)